docs/index.md

```diff
@@ -28,7 +28,7 @@ This tool is intended for use by developers who are experienced with JavaScript.
 ## Including the Buy SDK
 
 ```html
-<script src="http://sdks.shopifycdn.com/js-buy-sdk/latest/shopify-buy.polyfilled.globals.min.js"></script>
+<script src="http://sdks.shopifycdn.com/js-buy-sdk/v0/latest/shopify-buy.umd.polyfilled.min.js"></script>
```

Comment:
I definitely see the value in supporting bleeding per major version but I would totally use `/0.x.x/build-file-name.js` instead of messing with `latest` via `/v0/latest/build-file-name.js`. `0.x.x` means what we're trying to provide in *semver*.

*two $*